images/linux/Ubuntu1604-README.md

```diff
@@ -102,7 +102,7 @@
 - Google Cloud SDK 326.0.0
 - Hub CLI 2.14.2
 - Netlify CLI 3.5.0
-- oc CLI 4.6.15
+- OpenShift CLI (oc) 4.6.15
```

Comment:
Hello @pcarney8 ,
We mustn't modify md files manually. These files are autogenerated.
Instead - we should change https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1#L284

`OpenShift CLI 4.6.15` should work.